_layouts/default.html

```diff
@@ -51,12 +57,9 @@ <h1 class="noselect">South Eugene Robotics Team</h1>
             </li>
             {% endif %}
             {% endfor %}
-            <li style="height: 36px">
-                <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_blank" style="margin-top: 5px">
-                    <input type="hidden" name="cmd" value="_s-xclick">
-                    <input type="hidden" name="hosted_button_id" value="XAPA9VJHWAE94">
-                    <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif" border="0" name="submit" title="Donate with PayPal" alt="Donate with PayPal button">
-                    <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
+            <li>
+                <form name="donateform" action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_blank" style="margin: 0">
+                    <a href="javascript:submitDonateForm()" class="donate">Donate</a>
```

Comment:
If you make this a `<button type="submit">` or `<input type="submit">` you won't need the hacky JS on lines 3-8.

Reply:
lol thanks

Reply:
I improved it by using `onclick`, so that way I don't need to use a `script` tag. Changing it to a button would mean a lot of refactoring in the css, but with `a` as the element, it inherits all the styles of the navigation items.